test/unit/math/rev/mat/functor/integrate_ode_adams_prim_test.cpp

```diff
@@ -137,17 +137,20 @@ TEST(StanMathOde_integrate_ode_adams, error_conditions) {
                                        x_int_bad, 0, 1e-8, 1e-10, 1e6),
                    std::out_of_range, "vector");
 
-  EXPECT_THROW_MSG(integrate_ode_adams(harm_osc, y0, t0, ts, theta, x, x_int, 0,
-                                       -1, 1e-6, 10),
-                   std::invalid_argument, "relative_tolerance");
-
-  EXPECT_THROW_MSG(integrate_ode_adams(harm_osc, y0, t0, ts, theta, x, x_int, 0,
-                                       1e-6, -1, 10),
-                   std::invalid_argument, "absolute_tolerance");
-
-  EXPECT_THROW_MSG(integrate_ode_adams(harm_osc, y0, t0, ts, theta, x, x_int, 0,
-                                       1e-6, 1e-6, -1),
-                   std::invalid_argument, "max_num_steps");
+  EXPECT_THROW(integrate_ode_adams(harm_osc, y0, t0, ts, theta, x, x_int, 0, -1,
```

Comment:
please revert these tests back. Also, the `EXPECT_THROW_MSG` doesn't just print the message; it actually checks that there's a message with that string in the exception.